package.json

```
@@ -41,16 +41,16 @@
   },
   "peerDependencies": {
     "@mikro-orm/core": "^6.0.0 || ^6.0.0-dev.0",
-    "@nestjs/common": "^10.0.0",
-    "@nestjs/core": "^10.0.0"
+    "@nestjs/common": "^11.0.0",
+    "@nestjs/core": "^11.0.0"
```

this would be a breaking change, we want to stay compatible with v10, unless it would be too hard to achieve

```suggestion
    "@nestjs/common": "^10.0.0 || ^11.0.0",
    "@nestjs/core": "^10.0.0 || ^11.0.0"
```

---

will test this locally

---

fix pushed

---

i also removed the dev dependency upgrade to reduce scope of breaking stuff

---

> i also removed the dev dependency upgrade to reduce scope of breaking stuff

lets keep that so we can see if things actually break with the new version

---

missed the comment, done